test/persistence-hooks.test.js

```diff
@@ -4,9 +4,11 @@
 // License text available at https://opensource.org/licenses/MIT
 
 'use strict';
-var should = require('./init');
-var suite = require('loopback-datasource-juggler/test/persistence-hooks.suite.js');
+describe('persistence-hooks suite test', function() {
+  before(function() {
+    require('./init.js');
+  });
 
-suite(global.getDataSource(), should, {
-  replaceOrCreateReportsNewInstance: true,
+  require('loopback-datasource-juggler/test/persistence-hooks.suite.js');
```

Comment:
You are absoluetely right!
Weird thing is that it is working on 
MYSQL: 5.6.
OS: windows
on local machine, let's me push one more commit after re-testing.